SimpleBrowser/Browser.cs

```diff
@@ -47,7 +47,8 @@ public class Browser
         private readonly IWebRequestFactory _reqFactory;
         private readonly Dictionary<string, BasicAuthenticationToken> _basicAuthenticationTokens;
         private NameValueCollection _navigationAttributes = null;
-
+        private System.Security.Cryptography.X509Certificates.X509CertificateCollection _clientCertificates;

```

Comment:
Add System.Security.Cryptography.X509Certificates to the usings at the top of the file. Remove the namespace from here.